calitp/storage.py

```diff
@@ -99,17 +99,9 @@ def make_name_bq_safe(name: str):
     return str.lower(re.sub("[^\w]", "_", name))  # noqa: W605
 
 
-def prefix_bucket(bucket):
-    # TODO: use once we're in python 3.9+
-    # bucket = bucket.removeprefix("gs://")
-    bucket = bucket.replace("gs://", "")
-    return f"gs://test-{bucket}" if is_development() else f"gs://{bucket}"
-
-
-AIRTABLE_BUCKET = prefix_bucket("gs://calitp-airtable")
-SCHEDULE_RAW_BUCKET = prefix_bucket("gs://calitp-gtfs-schedule-raw")
-RT_RAW_BUCKET = prefix_bucket("gs://calitp-gtfs-rt-raw")
-SCHEDULE_PROCESSED_BUCKET = prefix_bucket("gs://calitp-gtfs-schedule-processed")
+AIRTABLE_BUCKET = os.getenv("CALITP_BUCKET__AIRTABLE")
+SCHEDULE_RAW_BUCKET = os.getenv("CALITP_BUCKET__SCHEDULE_RAW")
```

Comment:
Ok sorry I know you ran these by me but I missed it until I went to set one of these myself -- I would call it `CALITP_BUCKET__GTFS_SCHEDULE_RAW` i.e., specify `GTFS`, and same for RT immediately below